test/common-utils/scenarios.json

```diff
@@ -189,6 +189,22 @@
             }
         }
     },
+    "configure_zshrc_without_overwrite": {
+        "image": "ghcr.io/rails/devcontainer/images/ruby:3.2.5",
```

Comment:
```suggestion
        "image": "mcr.microsoft.com/devcontainers/ruby:3.2",
```

Reply:
We don't want our tests to depend on a third party and less well-known image as they have the potential of breaking on top of having security concerns.

Can we use a Dockerfile and recreate the scenario?

Reply:
I have discarded this scenario and created a new one as suggested by you.
name of the new scenario with Dockerfile is called `devcontainer-ruby-zshrc`
Kindly review the new code commit